### PR TITLE
Add Sagemaker Tensorflow Unet notebook

### PR DESCRIPTION
**Pull request by prateekch91**
Issue #, if available: N/A

Description of changes:
Adding Tensorflow 1.15.3 Unet notebooks to Studio and sagemaker_neo_compilation_jobs

Two examples were changed:
sagemaker-neo-tf-unet.ipynb - sagemaker_neo_compilation_jobs
sagemaker-neo-tf-unet.ipynb - aws_sagemaker_studio/sagemaker_neo_compilation_jobs

Testing:
Both notebooks run to completion successfully in SageMaker Instance and SageMaker Studio.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_predictor.delete_endpoint()"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (ngluna) [inline sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb:558]: Nit: Remove unnecessary cell.

File: aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+    "!wget {model_archive_url}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tar -xvzf unet-medical.tar.gz"
```
Comment (ngluna) [inline aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb:156]: Maybe add a short sentence saying something like: "The pre-trained model and its artifacts are saved in a compressed tar file (.tar.gz) so unzip first with:"

File: aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "RSxGrG4j-uxa"
+   },
+   "source": [
+    "## Download U-Net Model\n",
+    "\n",
+    "The SageMaker Neo TensorFlow Serving Container works with any model stored in TensorFlow's [SavedModel format](https://www.tensorflow.org/guide/saved_model). This could be the output of your own training job or a model trained elsewhere. For this example, we will use a pre-trained version of U-Net model based on this [repo](https://github.com/kamalkraj/DATA-SCIENCE-BOWL-2018)."
```
Comment (ngluna) [inline aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb:120]: Nit: Missing "the" 
 "For this example, we will use a pre-trained version of the U-Net..."

**Review by ngluna — COMMENTED** [review]
Very minor text changes. Other than that, looks good! Thanks for adding this notebook

**Comment by ngluna** [comment]
Notebooks are failing because CI is using the wrong kernel. Will override.

File: aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb (hunk 3 above)
Comment (prateekch91) [reply aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb]: Thanks for reviewing it. I have addressed all the comments.

File: aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb (hunk 2 above)
Comment (prateekch91) [reply aws_sagemaker_studio/sagemaker_neo_compilation_jobs/tensorflow_unet/sagemaker-neo-tf-unet.ipynb:163]: Addressed.

**Pull request merged**